tests/cloner_test.go

```diff
@@ -750,6 +751,104 @@ var _ = Describe("Namespace with quota", func() {
 	})
 })
 
+var _ = Describe("[rfe_id:1277][crit:high][vendor:cnv-qe@redhat.com][level:component]Cloner Test Suite", func() {
+	f := framework.NewFrameworkOrDie(namespacePrefix)
+
+	var sourcePvc *v1.PersistentVolumeClaim
+	var targetPvc *v1.PersistentVolumeClaim
+
+	AfterEach(func() {
+		if sourcePvc != nil {
+			By("[AfterEach] Clean up source PVC")
+			err := f.DeletePVC(sourcePvc)
+			Expect(err).ToNot(HaveOccurred())
+		}
+		if targetPvc != nil {
+			By("[AfterEach] Clean up target PVC")
+			err := f.DeletePVC(targetPvc)
+			Expect(err).ToNot(HaveOccurred())
+		}
+	})
+
+	It("[test_id:3999] Create a data volume importing an image and then clone it and verify retry count", func() {
+		pvcDef := utils.NewPVCDefinition(sourcePVCName, "1G", nil, nil)
+		pvcDef.Namespace = f.Namespace.Name
+		sourcePvc = f.CreateAndPopulateSourcePVC(pvcDef, sourcePodFillerName, fillCommand+testFile)
+		targetNs, err := f.CreateNamespace(f.NsPrefix, map[string]string{
```

Comment:
This is more an FYI than an issue. The targetNS and addNamespaceToDelete are for when you are cloning across namespaces. This particular test we don't care about that and could have used the same namespace and saved us writing some code.

Reply:
I thought that smart clone might be used if I do not use two distinct ns, and then I'll not be able to break the pod.

But probably not.

Reply:
That is a good thought, but we have lanes that do not use smart clone and we would be testing in those lanes. But this test would fail in the lanes that do use smart clone, so we would have to skip the test in those lanes. We have tests we skip already in that particular case.